server/routes/api/chart.py

```diff
@@ -378,3 +378,164 @@ def geojson(dcid):
     }),
                     200,
                     mimetype='application/json')
+
+
+def get_choropleth_sv():
+    """ Gets all the stat vars and accompanying denominators that will have a choropleth chart drawn for them
+    ie. charts with the hasChoropleth property set as true
+
+    Returns: 
+        a tuple consisting of:
+            set of all stat vars including both the stat vars of the charts and stat vars of denominators
+            dictionary of stat var: list of denominators
+    """
+    chart_config = current_app.config['CHART_CONFIG']
+    if os.environ.get('FLASK_ENV') == 'development':
+        with bp.open_resource('../../chart_config.json') as f:
+            raw_config = json.load(f)
+    all_sv = set()
+    sv_denom_mapping = {}
+    for config in chart_config:
+        if 'hasChoropleth' in config and config['hasChoropleth']:
+            # we should only be making choropleths for configs with a single stat var
+            sv = config['statsVars'][0]
+            all_sv.add(sv)
+            denom = ''
+            if 'denominator' in config:
+                denom = config['denominator'][0]
+                all_sv.add(denom)
+            elif 'perCapita' in config and config['perCapita']:
+                # The denominator used for perCapita is total population
+                all_sv.add('Count_Person')
+                denom = 'Count_Person'
+            if sv not in sv_denom_mapping:
+                sv_denom_mapping[sv] = set()
+            sv_denom_mapping[sv].add(denom)
+    return all_sv, sv_denom_mapping
+
+
+def get_data_for_statvar(sv, geos, all_sv_data):
```

Comment:
would be good to also return the provenance url which would be used in the chart footnote. can add a TODO for that.